mule-extensions-api-dsql/src/main/java/org/mule/runtime/extension/api/dsql/DsqlParser.java

```diff
@@ -9,8 +9,8 @@
 import org.mule.runtime.extension.internal.MuleDsqlLexer;
 import org.mule.runtime.extension.internal.MuleDsqlParser;
 import org.mule.runtime.extension.internal.exception.DsqlParsingException;
-import org.mule.runtime.extension.internal.grammar.DefaultDsqlGrammarVisitor;
 import org.mule.runtime.extension.internal.grammar.BaseDsqlNode;
+import org.mule.runtime.extension.internal.grammar.DefaultDsqlGrammarVisitor;
```

Comment:
check your IDE configuration. Looks like the import ordering is wrong, generating false changes such as these

Reply:
Oh, it was not the IDE config in this case. I've run the Impsort maven plugin, configured according to the "Mule Runtime - PR guidelines".